telegramBot.js

```diff
@@ -0,0 +1,200 @@
+            const helpText = `You can control me by sending these commands:\n
+                                \t/init - initialize bot\n
+                                \t/status - check status of bot\n
+                                \t/gm <platform> <message> - sends text message to chosen platform\n`
+
+            ctx.reply(helpText)
+        })
+
+        this.bot.command('quit', (ctx) => {
+            ctx.leaveChat()
+        })
+        
+        this.bot.command('gm', async (ctx) => {
+            if(this.initialzied) {
+
+                if(this.onCommandReceived){
+                    const input = ctx.message.text.slice(1).trim().split(' ');
+                    const command = input.shift();
+                    const commandArgs = input.join(' ');
+                    const result = await this.onCommandReceived(command, commandArgs, 'telegram');
+                    console.log(result);
+                    return result;
+                }
+                
+            } else {
+                ctx.reply('Bot uninitialized. Run /init command to initialize.')
+            }
+        })
+        
+        this.bot.command('fs', async (ctx) => {
+            if(this.initialzied)  {
+                if(this.onCommandReceived){
+                    const input = ctx.message.text.slice(1).trim().split(' ');
+                    const command = input.shift();
+                    const commandArgs = input.join(' ');
+                    const result = await this.onCommandReceived(command, commandArgs, 'telegram');
+                    console.log(result);
+                    return result;
+                }
+            
+                const fileURL = "http://i.imgur.com/5rOhtdL.png"
+                this.sendFile(fileURL)
+            } else {
+                ctx.reply('Bot uninitialized. Run /init command to initialize.')
+            }
+        })
+
+        this.bot.command('fs_r', async (ctx) =>{
+            if(this.initialzied)  {
+                if(this.onCommandReceived){
+                    const input = ctx.message.text.slice(1).trim().split(' ');
+                    const command = input.shift();
+                    const commandArgs = input.join(' ');
+                    const result = await this.onCommandReceived(command, commandArgs, 'telegram');
+                    console.log(result);
+                    return result;
+                }
+            
+                const fileURL = 'https://homepages.cae.wisc.edu/~ece533/images/airplane.png'
+                this.sendFileRemote(fileURL)
+            } else {
+                ctx.reply('Bot uninitialized. Run /init command to initialize.')
+            }
+        })
+
+        this.bot.use((ctx) => {
+            this.chatID = ctx.chat.id
+            this.chatTitle = ctx.chat.title
+        })
+
+    }
+
+    setCommandListener(commandListener) {
+        this.onCommandReceived = commandListener;
+    }
+
+    async getUpdates(){
```

Comment:
could you put some comments why we need this function ?

Reply:
I've removed it